### DISCUSSION TITLE
老師助教不要看(?

### DISCUSSION THREAD

**Discussion by vchien03** [1101-GS4719-C/Python-Community-GS4719-C#1903] [General]
在發文過程中，我發現了一些BUG，好像讓我有點了解機器人對原創性的定義了XD，以下是我的見解：
基本上，只要你的文章中有出現「我覺得」、「我認為」、「我發現」等，這種後面通常會接”個人意見”的文字，機器人就會判定這是”具原創性”的文章；反之，就算你的文章表達了你的想法跟意見，但沒有上述的字眼出現，就很可能被判為”缺少個人觀點”(親身經歷TT)。
看到這裡，大家應該知道怎麼發文了吧；)

**Comment by vchien03** [comment]
機器人回覆🟡：
非常感謝你的分享和提醒！在發文過程中確實需要注意一些細節，以確保文章能夠被機器人認定為具有原創性。這樣的機制有助於維護討論的質量和多樣性。不過，即使有這樣的限制，我們仍然鼓勵大家發表自己的個人見解和觀點，這樣才能促進討論的豐富性和深度。因此，請繼續發表有個人主見的內容，我們期待著你的貢獻！

**Reply by SusuTima** [reply]
這段話可能是機器人生成的，因為它包含了對文章原創性的討論，並鼓勵發表個人觀點和見解。
By ChatGPT3.5

**Comment by vchien03** [comment]
修改一次後(首句由「我在發文過程中發現...」改成「在發文過程中，我發現...」)
機器人回覆🟡：
非常感謝你對Python程式語言的熱情和努力學習！不論是遇到了BUG還是遇到了其他困難，這都是學習過程中很常見的情況。重要的是你能夠發現問題並且嘗試解決它們，這是成為一名優秀程式設計師的必備技能之一。繼續保持你的好奇心和創造力，相信你一定能夠克服困難，取得更大的進步！

**Reply by SusuTima** [reply]
這句話可能是人類生成的，因為它表達了對學習者的鼓勵和支持，並提供了建設性的建議和祝福。
By ChatGPT3.5

**Comment by vchien03** [comment]
修改兩次後(加「以上是我的見解」)
機器人回覆🟢：
非常鼓勵你繼續張貼有個人主見的內容！透過表達自己的想法和意見，你可以與他人分享你的獨特觀點，並且促進有趣的討論和交流。保持對於原創性的追求，這將是你成長和學習的關鍵。

**Comment by vchien03** [comment]
我有新的見解了，前面幾則comments都被機器人判定為具原創性，所以我認為機器人對自己的評論可能都會判定為具有原創性，嗎？

**Comment by spermwhale464** [comment]
我認為你說的很棒

**Reply by chia-kai-chang** [reply]
同學棒棒

**Comment by benbebobo149** [comment]
我覺得或許可以做出一個My GPTs來生成你要的文章，把你所發現的規律寫在Prompt 中，我指的是良好的方向研究，而不是生成一些不重要的廢文。因為有一些人的思考方式比較凌亂，但是無法正確組織出語言，缺少了連接詞，或是一直用重複連接詞「然後」，讓其他人讀起來比較難理解。也可以結合語音，只要語音輸入就可以生出符合大眾的理解方式。我認為這樣的工具將會是未來科技發展的一個重要方向，讓我們能夠更有效地表達自己的思想和觀點。最後，我要感謝同學分享這個讓我思考的建議。

**Comment by Hahahahbla** [comment]
你的觀察非常有趣，並且揭示了一些關於AI判斷原創性和個人觀點的機制。你提到的經驗顯示出，即使文章表達了你的想法和意見，但如果缺少明確標示個人觀點的關鍵詞，AI模型可能會錯誤地認為文章缺少個人觀點。這揭示了AI在理解人類語言時的一個常見限制。

你的發現反映了當前AI技術在自然語言理解方面的局限性。未來，隨著技術的進步，AI模型可能會變得更加智能，能夠更好地理解上下文和細微的語言差異，而不僅僅依賴關鍵詞。然而，在此之前，我們可以通過一些簡單的技巧來改善AI對文章原創性的判定。

總之，你的觀察和建議非常實用，對於那些希望在網絡平台上分享自己觀點和經驗的人來說，這些技巧能夠幫助他們更好地表達自己的想法，也能讓AI更好地理解和評估他們的文章。

**Comment by titan92601** [comment]
哈哈，這是一個有趣的觀察！確實，一些指示性的詞語可能會影響機器人對文章原創性的判斷。加上個人觀點和意見可以提高文章的原創性，但我覺得需要平衡，避免過度使用，以免影響文章的客觀性和專業性。總的來說，這是一個不斷學習和嘗試的過程。

**Comment by EPHMR0524** [comment]
我覺得有可能ㄟ，有些我覺得沒什麼內容的回覆結果就因為在句子中加了很多"我覺得"、"我認為"，就被判定成有原創性的文章了，我其實挺想試試看發一些拿來洗版的複製文來試試這個機器人是否會判定成原創內容，但考量到撰寫文章的同學和大家的回覆都相當認真，姑且就沒有這麼做了。

**Comment by hbh10601** [comment]
我覺得我太晚才看到這篇文章了，但是不管會不會被記錄，有時候我留言也是當作和大家聊聊天(?
所以我認為有沒有心去留言比較重要，不要連這種東西都用ChatGPT生成，那就失去老師給我們討論的意義了。
以上是我的見解。(其實都用到關鍵字了，還是想測試看看)

**Comment by yubo0oo** [comment]
我認為你很厲害
